src/PrestaShopBundle/Resources/config/routing/admin/improve/international/tax.yml

```diff
@@ -0,0 +1,13 @@
+admin_taxes_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Improve\International\Tax:index'
+    _legacy_controller: AdminTaxes
+
+admin_taxes_save_options:
+  path: /
```

Comment:
i think having `/save-options` path would be better, since grid is using `POST /` for filtering in other PR.